pageserver/src/tenant/storage_layer/delta_layer.rs

```diff
@@ -955,15 +955,15 @@ impl DeltaLayerInner {
         }
     }
 
-    pub(super) async fn load_val_refs<T: AsRef<DeltaLayerInner> + Clone>(
-        this: &T,
-    ) -> Result<Vec<(Key, Lsn, ValueRef<T>)>> {
+    pub(super) async fn load_val_refs(
+        this: &Arc<DeltaLayerInner>,
+    ) -> Result<Vec<(Key, Lsn, ValueRef<Arc<DeltaLayerInner>>)>> {
```

Comment:
Please don't do this, my PR depends on this: #4938. I can't even get a good test run from all these conflicts.

Reply:
This should be gone with the latest merge from `main`

Reply:
I think it is gone.